irrep/__kpoint.py

```diff
@@ -341,7 +344,7 @@ def write_characters(self,degen_thresh=1e-8,irreptable=None,symmetries=None,prel
         print ("\n\nk-point {0:3d} :{1} \n number of irreps = {2}".format(self.ik0,self.K,Nirrep) )
         print ("   Energy  | multiplicity |{0} irreps {0}| sym. operations  ".format(s2))
         print ("           |              |{0}        {0}| ".format(s2)," ".join(s1+"{0:4d}    ".format(i)+s1 for i in sorted(sym)))
-        print ("\n".join( (" {0:8.4f}  |    {1:5d}     | {2:"+str(irreplen)+"s} |").format(e-efermi,d,ir)+
+        print ("\n".join( (" {0:8.4f}  |    {1:5d}     | {2:"+str(irreplen)+"s} |").format(e-efermi*Ry2eV,d,ir)+
```

Comment:
In this function all energies are assumed in eV, including efermi, and it does not depend on the code. Better apply units conversion in the babdstructure.__init_espresso  method